src/GUniLPOracle.t.sol

```diff
@@ -586,4 +592,24 @@ contract GUniLPOracleTest is DSTest {
         assertLt(sqrtPriceX96, 1 << 160);
     }
 
+    function test_sqrt_price_ratio_fuzz(uint256 p0, uint256 dec0, uint256 p1, uint256 dec1) public {
+        p0 %= MAX_PRICE;
+        if (p0 < WAD) p0 = MIN_PRICE;
+        if (p0 > MAX_PRICE) p0 = MAX_PRICE;
```

Comment:
mmm just wondering - since we have the modulu do we really need this trimming again? relevant for other places as well.

Reply:
Good point.
Regarding the case `p0 < WAD` we want to make sure that the price is not too low otherwise the sqrtPX96 formula will return zero.
Done in e7470e5.